tests/api/test_api_clients.py

```diff
@@ -12,22 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import datetime
 import json
 import time
 import uuid
 import unittest
 import zlib
 
 import diskcache
+import http_sf
```

Comment:
I am sure you thought about it but `http_sf` does not seem like it has a tonne of adoption. I think there is some risk that it gets abandoned. Can always adjust later but :shrug: